test/functional/test_framework/test_framework.py

```diff
@@ -675,19 +676,20 @@ def _initialize_chain(self):
             # Set a time in the past, so that blocks don't end up in the future
             cache_node.setmocktime(cache_node.getblockheader(cache_node.getbestblockhash())['time'])
 
-            # Create a 199-block-long chain; each of the 4 first nodes
-            # gets 25 mature blocks and 25 immature.
-            # The 4th node gets only 24 immature blocks so that the very last
-            # block in the cache does not age too much (have an old tip age).
-            # This is needed so that we are out of IBD when the test starts,
-            # see the tip age check in IsInitialBlockDownload().
-            for i in range(8):
-                cache_node.generatetoaddress(
-                    nblocks=25 if i != 7 else 24,
-                    address=TestNode.PRIV_KEYS[i % 4].address,
-                )
-
-            assert_equal(cache_node.getblockchaininfo()["blocks"], 199)
+            if self.chain != "signet":
```

Comment:
why is this needed when setup_clean_chain is set to true?

Reply:
Because `cache_node` is perhaps unable to generate any blocks at all, due to lacking the privkeys to properly sign them.

Reply:
locally it is passing for me. What is the error message you get? This code path should only be executed when `setup_clean_chain==False`

Reply:
Ahh I see what you're saying. Removing this part, thanks!